### ISSUE TITLE
Operator crash

### ISSUE THREAD

**Issue by wcpettus** [project8/swarm#7]
swarm/operator crashed today at 10:26 after running smoothly for ~2.5 days

stderr logfile shows:
```
panic: runtime error: index out of range

goroutine 1 [running]:
panic(0x947da0, 0xc82000e030)
    /usr/local/go/src/runtime/panic.go:464 +0x3e6
main.main()
    /home/project8/python_environments/latest_version/src/github.com/project8/swarm/Go/operator/Operator.go:323 +0x397d
```

stdout logfile shows:
```
1e10 10:21:49.589 INFO [main] ▶ Attempting to notify the operator
1e11 10:22:02.854 INFO [main] ▶ Current latency: 100.276322ms
1e12 10:22:32.852 INFO [main] ▶ Current latency: 98.318832ms
1e13 10:23:02.855 INFO [main] ▶ Current latency: 101.183237ms
1e14 10:23:32.852 INFO [main] ▶ Current latency: 98.500827ms
1e15 10:24:02.853 INFO [main] ▶ Current latency: 99.99386ms
1e16 10:24:32.852 INFO [main] ▶ Current latency: 98.45886ms
1e17 10:25:02.852 INFO [main] ▶ Current latency: 98.106984ms
1e18 10:25:32.854 INFO [main] ▶ Current latency: 100.070887ms
1e19 10:26:02.852 INFO [main] ▶ Current latency: 98.789373ms
1e1a 10:26:32.852 INFO [main] ▶ Current latency: 98.723291ms
1e1b 10:26:38.169 INFO [func1] ▶ Disconnecting from RTM
```

Not sure why the counter would be an index out of range at 1e1b, the last "attempting to notify the operator" was a successful @operator call in Slack.

Restarting the service again to see what we get.

**Comment by nsoblath** [comment]
I think the problem is an assumption that the message text is a non-empty string. I'll fix this tomorrow. As a reminder to myself, the problem is now at line 472 in the current master branch (line 323 at the time of the crash)

**Comment by nsoblath** [comment]
This has been fixed in v1.4.1
